### ISSUE TITLE
ERROR: lstat /var/lib/teleport/host_uuid: no such file or directory

### ISSUE THREAD

**Issue by sharadregoti** [gravitational/teleport#8077]
### Description
I am following this doc https://goteleport.com/docs/database-access/guides/postgres-self-hosted/ to setup self hosted postgres.
The docs says to run this command `tctl auth sign --format=db --host=db.example.com --out=server --ttl=2190h
TTL`

As I didn't had `tctl`, I tried installing it using the following link https://get.gravitational.com/teleport-v7.1.0-linux-amd64-bin.tar.gz

After unzipping the file I ran the install command `sudo ./install` 
Which gave the following output
```
Teleport binaries have been copied to /usr/local/bin

Thanks for installing Teleport.

Is it OK if we collect some info about your install?
Please run this command to send in a survey.
Optional: Replace email to join our newsletter and get a swag package.
$ curl -X POST https://usage.teleport.dev -F OS=linux -F use-case="access my ..." -F email="alice@example.com"

Otherwise, ignore!
```

The I tried running the doc command
`tctl auth sign --format=db --host=ec2-65-2-34-23.ap-south-1.compute.amazonaws.com --out=server --ttl=2190h`

It gave me this error
`ERROR: lstat /var/lib/teleport/host_uuid: no such file or directory`

**What happened**:
An error occured.

**What you expected to happen**:
I expected the command would generate the appropriate certificates

### Reproduction Steps
As minimally and precisely as possible, describe step-by-step how to reproduce the problem.
1.
2.
3.

### Server Details
- Teleport version (run `teleport version`): 7.1
- Server OS (e.g. from `/etc/os-release`):  Amazon Linux
- Where are you running Teleport? (e.g. AWS, GCP, Dedicated Hardware): AWS EC2
- Additional details:

### Client Details
- Tsh version (`tsh version`): 7.1
- Computer OS (e.g. Linux, macOS, Windows): Linux
- Browser version (for UI-related issues):
- Installed via (e.g. apt, yum, brew, website download): website download
- Additional details:

### Debug Logs
Please include or attach debug logs, when appropriate. Obfuscate sensitive information!
- Start Teleport with --debug flag (`teleport --debug`)
- Run tsh with --debug flag (`tsh --debug`)

```
DEBU             Debug logging has been enabled. common/tctl.go:282
DEBU             No config file or identity file, loading auth config via extension. common/tctl.go:308

ERROR REPORT:
Original Error: *trace.NotFoundError lstat /var/lib/teleport/host_uuid: no such file or directory
Stack Trace:
        /go/src/github.com/gravitational/teleport/lib/utils/utils.go:324 github.com/gravitational/teleport/lib/utils.ReadPath
        /go/src/github.com/gravitational/teleport/lib/utils/utils.go:431 github.com/gravitational/teleport/lib/utils.ReadHostUUID
        /go/src/github.com/gravitational/teleport/tool/tctl/common/tctl.go:355 github.com/gravitational/teleport/tool/tctl/common.applyConfig
        /go/src/github.com/gravitational/teleport/tool/tctl/common/tctl.go:147 github.com/gravitational/teleport/tool/tctl/common.Run
        /go/src/github.com/gravitational/teleport/tool/tctl/main.go:37 main.main
        /opt/go/src/runtime/proc.go:225 runtime.main
        /opt/go/src/runtime/asm_amd64.s:1371 runtime.goexit
User Message: lstat /var/lib/teleport/host_uuid: no such file or directory
```

**Comment by sharadregoti** [comment]
Yes, it started working after I started teleport. 

I think the docs can be updated (https://goteleport.com/docs/database-access/guides/postgres-self-hosted/#create-certificatekey-pair) to specify that teleport needs to be up & running before creating the secret.

**Comment by zmb3** [comment]
This was resolved with #13975
